crates/common/trie/benches/trie.rs

```diff
@@ -0,0 +1,207 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::sync::Mutex;
+
+use criterion::{criterion_group, criterion_main, Criterion};
+
+use hasher::HasherKeccak;
+use uuid::Uuid;
+
+use cita_trie::{MemoryDB as CitaMemoryDB, PatriciaTrie as CitaPatriciaTrie, Trie as CitaTrie};
+
+use ethrex_trie::{InMemoryTrieDB as EthrexMemoryDB, Trie as EthrexTrie};
+
+fn cita_new_trie() -> CitaPatriciaTrie<CitaMemoryDB, HasherKeccak> {
+    CitaPatriciaTrie::new(
+        Arc::new(CitaMemoryDB::new(false)),
+        Arc::new(HasherKeccak::new()),
+    )
+}
+
+fn ethrex_new_trie() -> EthrexTrie {
+    let hmap: HashMap<Vec<u8>, Vec<u8>> = HashMap::new();
+    let map = Arc::new(Mutex::new(hmap));
+    let db = EthrexMemoryDB::new(map);
+    EthrexTrie::new(Box::new(db))
+}
+
+fn cita_trie_benchmark(c: &mut Criterion) {
```

Comment:
Let's keep the ethrex first, since it's the one we really care about

Reply:
do you want me to remove cite benchmarks?